README.md

```diff
@@ -464,6 +464,35 @@ withPipeline(type, product, component) {
 
 ```
 
+## Import terraform modules created using template deployment to native Terraform resources
+#### Usage
+
+Terraform AzureRM provider now supports new resource types, which were previously created using Azure Template Deployment.
+
+Currently, resources created using the following modules can be imported:
+
+* Service Bus Namespace (https://github.com/hmcts/terraform-module-servicebus-namespace)
+* Service Bus Topic (https://github.com/hmcts/terraform-module-servicebus-topic)
+* Service Bus Queue (https://github.com/hmcts/terraform-module-servicebus-queue)
+* Service Bus Subscription (https://github.com/hmcts/terraform-module-servicebus-subscription)
+
+RPE have released new versions of these modules, where native terraform resource types are used. The new version is available in a separate branch in the respective repositories.
+
+To consume the new modules, existing resources must be imported to the new module structure. This can be achieved by updating the branch reference as shown in the Git Commit below and using the `importTerraformModules()` method in Application and Infrastructure pipelines. It is advisable to run the pipeline with this method call once. Subsequent runs will create an empty stage, which can be avoided.
```

Comment:
Does this document needs refresh ? As it is no more optional ?

Reply:
Updated